adapters/json/json.go

```diff
@@ -0,0 +1,139 @@
+package json
+
+import (
+	"encoding/json"
+	"errors"
+	"log"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"github.com/gliderlabs/logspout/adapters/syslog"
```

Comment:
I think separation of packages is of highest priority for a maintainable system, but as noted below it can be solved by moving/renaming the syslog message to some kind of base message where it's clear that it's supposed to be used by several adapters.